R/release.R

```diff
@@ -193,25 +193,32 @@ use_github_release <- function(host = deprecated(),
   kv_line("Tag name", tag_name)
   kv_line("SHA", dat$SHA)
 
+  if (git_can_push()) {
+    git_push()
+  }
   check_github_has_SHA(SHA = dat$SHA, tr = tr)
 
   news <- get_release_news(SHA = dat$SHA, tr = tr)
 
   gh <- gh_tr(tr)
+
+  cli::cli_inform("Publishing {tag_name} release to GitHub")
   release <- gh(
     "POST /repos/{owner}/{repo}/releases",
-    name = release_name, tag_name = tag_name,
-    target_commitish = dat$SHA, body = news, draft = TRUE
+    name = release_name,
+    tag_name = tag_name,
+    target_commitish = dat$SHA,
+    body = news,
+    draft = draft
   )
+  cli::cli_inform("Release at {.url {release$html_url}}")
 
   if (!is.null(dat$file)) {
-    ui_done("{ui_path(dat$file)} deleted")
+    cli::cli_inform("Deleting {.path {dat$file}}")
     file_delete(dat$file)
   }
 
-  Sys.sleep(1)
-  view_url(release$html_url)
```

Comment:
Do we still want to open the page?

Reply:
I don't think it's needed because the link we print is clickable.